src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotNmethod.java

```diff
@@ -78,52 +78,17 @@ boolean inOopsTable() {
     private final long compileIdSnapshot;
 
     /**
-     * This is a mirror of the ChangeReason enum defined in nmethod.hpp.
-     * It defines constants representing different reasons why an nmethod
-     * changed or was invalidated.
+     * Identify the reason that caused this nmethod to be invalidated.
```

Comment:
Update doc to note that the field will be `-1` until the nmethod is invalidated.